execution_workflows/APA-Scan/workflow/Snakefile

```diff
@@ -0,0 +1,357 @@
+"""
```

Comment:
Would be good to have the Snakemake experts review this, e.g., @mkatsanto, @dominikburri, @ninsch3000